packages/story-editor/src/components/canvas/displayElement.js

```diff
@@ -106,6 +108,15 @@ function DisplayElement({ element, previewMode, isAnimatable = false }) {
     getBox: state.actions.getBox,
     dataToEditorX: state.actions.dataToEditorX,
   }));
+  const { getProxiedUrl } = useCORSProxy();
```

Comment:
Would be interesting to know if/how these changes influence re-rendering / performance. Since the dependencies are now higher up in the hierarchy, wondering if this can also influence performance.

Reply:
Tried one solution which prevents re-rendering when tested, can you take a look https://github.com/GoogleForCreators/web-stories-wp/pull/10452/commits/1767cf0c066c560c4148881260ab8c26cf8d1ace